management/http.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	msVersionHeader           = "x-ms-version"
-	msVersionHeaderValue      = "2014-05-01"
+	msVersionHeaderValue      = "2014-10-01"
```

Comment:
is there a reason to update this? if not let's leave it as is. in fact, this value should come from the `Config` object we added very recently.